determine-current-version.sh

```diff
@@ -15,6 +15,8 @@ if [ "$PROJECT" == "orm" ]; then
 elif [ "$PROJECT" == "reactive" ]; then
 	# For example, if `version.properties` contains `projectVersion=2.4.5-SNAPSHOT`, it returns `2.4.5-SNAPSHOT`
 	grep projectVersion $WORKSPACE/gradle/version.properties|cut -d'=' -f2
+elif [ "$PROJECT" == "models" ]; then
+	cat $WORKSPACE/version.txt
```

Comment:
Interesting, I would have expected this to be in line with our other gradle projects...

Reply:
yeah ... I'm not 100% convinced that this is correct 🙈 but ... for models https://github.com/hibernate/hibernate-models/blob/10a7869006900710d67d4ac92ac7f67e2c4c168e/version.txt#L1 that's where the version seems to live

Reply:
Maybe worth pinging Steve? We can certainly consider aligning hibernate-models on other projects, or vice-versa.

Reply:
Consistency is probably good here.  Personally not a fan of the `gradle/version.properties` location, but that's definitely the path of least resistance.